Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN GOOS=${TARGETOS} GOARCH=${TARGETARCH} make aws-efs-csi-driver
 
 FROM amazonlinux:2.0.20200602.0
```

Comment:
Should we use latest tag, would it have the newest versions of these packages?

Reply:
We can push the change for `amazonlinux:2.0.20210219.0`. This tag does have the packages to be addressed in the vulnerabilities. We don't know if the latest tag will introduce any other vulnerabilities. Is that okay ?

Reply:
@amitkc1  yep, I'll investigate using latest tag separately: https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/353

There are apaprently some other CVEs and I want to squash as many as possible but, as you say, we don't know if latest will introduce more. https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/343